src/Symfony/RequiredAutowiringExtension.php

```diff
@@ -0,0 +1,95 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Symfony;
+
+use PHPStan\BetterReflection\Reflection\Adapter\ReflectionClass;
+use PHPStan\Reflection\AdditionalConstructorsExtension;
+use PHPStan\Reflection\ClassReflection;
+use PHPStan\Reflection\Php\PhpPropertyReflection;
+use PHPStan\Reflection\PropertyReflection;
+use PHPStan\Rules\Properties\ReadWritePropertiesExtension;
+use PHPStan\Type\FileTypeMapper;
+use Symfony\Contracts\Service\Attribute\Required;
+use function class_exists;
+use function count;
+
+class RequiredAutowiringExtension implements ReadWritePropertiesExtension, AdditionalConstructorsExtension
+{
+
+	/** @var FileTypeMapper */
+	private $fileTypeMapper;
+
+	public function __construct(FileTypeMapper $fileTypeMapper)
+	{
+		$this->fileTypeMapper = $fileTypeMapper;
+	}
+
+	public function isAlwaysRead(PropertyReflection $property, string $propertyName): bool
+	{
+		return false;
+	}
+
+	public function isAlwaysWritten(PropertyReflection $property, string $propertyName): bool
+	{
+		return false;
+	}
+
+	public function isInitialized(PropertyReflection $property, string $propertyName): bool
+	{
+		// If the property is public, check for @required on the property itself
+		if (!$property->isPublic()) {
+			return false;
+		}
+
+		if ($property->getDocComment() !== null && $this->isRequiredFromDocComment($property->getDocComment())) {
+			return true;
+		}
+
+		// Check for the attribute version
+		if (class_exists(Required::class) && $property instanceof PhpPropertyReflection && count($property->getNativeReflection()->getAttributes(Required::class)) > 0) {
+			return true;
+		}
+
+		return false;
+	}
+
+	public function getAdditionalConstructors(ClassReflection $classReflection): array
+	{
+		$additionalConstructors = [];
+		/** @var ReflectionClass $nativeReflection */
```

Comment:
Please remove the inline `@var`.

Reply:
On PHP 7.3 and lower this will throw the following;
`Call to method getMethods() on an unknown class PHPStan\BetterReflection\Reflection\Adapter\ReflectionEnum.`
I'm not quite sure what to do with this, as I'm confused as to why it would do this only on PHP 7.3 and lower